File: app/src/main/java/com/xuvjso/nfmovies/Fragment/MoreFragment.java
package com.xuvjso.nfmovies.Fragment;


import android.os.Bundle;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import com.xuvjso.nfmovies.R;
import me.yokeyword.fragmentation.SupportFragment;

/**
 * A simple {@link Fragment} subclass.
 * Use the {@link MoreFragment#newInstance} factory method to
 * create an instance of this fragment.
 */
public class MoreFragment extends SupportFragment {

    public MoreFragment() {
        // Required empty public constructor
    }

    public static MoreFragment newInstance() {
        MoreFragment fragment = new MoreFragment();
        return fragment;
    }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_more, container, false);
        getChildFragmentManager().beginTransaction().replace(R.id.setting_container, new SettingFragment()).commit();
        return view;
    }
}

File: app/src/main/java/com/futuretech/nfmovies/Fragment/MoreFragment.kt
package com.futuretech.nfmovies.Fragment


import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.futuretech.nfmovies.R
import me.yokeyword.fragmentation.SupportFragment

/**
 * A simple [Fragment] subclass.
 * Use the [MoreFragment.newInstance] factory method to
 * create an instance of this fragment.
 */
class MoreFragment : SupportFragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_more, container, false)
        childFragmentManager.beginTransaction().replace(R.id.setting_container, SettingFragment()).commit()
        return view
    }

    companion object {

        fun newInstance(): MoreFragment {
            return MoreFragment()
        }
    }
}// Required empty public constructor
